templates/course/lesson-count.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Lessons count in a Course.
+ *
+ * @package LifterLMS/Templates
+ *
+ * @since [version]
+ * @version [version]
+ *
+ * @author      LifterLMS
+ * @package     LifterLMS/Templates
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+global $post;
+
+$course = new LLMS_Course( $post );
+$lessons_count = $course->get_lessons_count();
+?>
+
+<div class="llms-meta llms-lessons-count">
+	<p><?php printf( __( 'Number of Lessons: <span class="lessons-count">%u</span>', 'lifterlms' ), $lessons_count ); ?></p>
```

Comment:
I would rather avoid putting html into translatable strings, you know what I mean ;)

Reply:
I've copied the logic from https://github.com/gocodebox/lifterlms/blob/0e342f5cc7fcf4f64f4b584b6c8f0d1471f65f6a/templates/course/length.php

what do you suggest, how should I use the span tag?

Reply:
as we discussed on slack, please refer to what you already did for the advanced videos :D

Reply:
thank you 👍